src/main/java/edu/harvard/iq/dataverse/settings/JvmSettings.java

```diff
@@ -105,7 +105,11 @@ public enum JvmSettings {
     SCOPE_PID_HANDLENET_KEY(SCOPE_PID_HANDLENET, "key"),
     HANDLENET_KEY_PATH(SCOPE_PID_HANDLENET_KEY, "path", "dataverse.handlenet.admcredfile"),
     HANDLENET_KEY_PASSPHRASE(SCOPE_PID_HANDLENET_KEY, "passphrase", "dataverse.handlenet.admprivphrase"),
-    
+
+    // FRONTEND
+    SCOPE_FRONTEND(PREFIX, "frontend"),
+    FRONTEND_URL(SCOPE_FRONTEND, "url"),
```

Comment:
Question: is it necessary to create another scope and do you expect more settings to be added here? From the docs and descriptions I understand this URL is only used for CORS so far, so maybe we could move this to the API scope?

Reply:
Should this setting be front-end-specific at all? If someone creates another client, would this need to be more than one url? I.e. should this be named something like ALLOWED_CORS_ORIGINS with a value that is a list of urls, defaulting perhaps to just the front end url?

Reply:
This sounds reasonable (and is easy to apprehend with the lookup function, just give it the `String[].class` as parameter...)

Should this include migrating `:AllowCors` to a or this JvmSetting? The list could default to "*" as was discussed above. (And probably there is no need for backward compatibility for a DB setting that defaulted to true.)

Reply:
@qqmyers I prefer to make it front-end specific, just for the Dataverse Frontend SPA. Making it configurable using that list of allowed origins may end up with someone turning on the feature flag to add their own clients and authenticate them via session cookie, while the only intended use case for the feature flag is SPA development. Remember the security risks and the temporary lifetime of the feature flag (It will be removed when we have a final auth mechanism).

@poikilotherm Personally I have no preference in one scope or another. I can move it to the API scope if it is clearer in that way. Ideally, I would have liked to add it to the same scope as the feature flag, because they depend on each other. But the current feature flag mechanism does not support that kind of "subproperties" (If I am not wrong.)